docs/how-to-guides/how-to-create-maps-for-autoware.md

```diff
@@ -0,0 +1,59 @@
+
+Traditionally, a Mobile Mapping System (MMS) is used in order to create highly accurate large-scale point cloud maps. However, since a MMS requires high-end sensors for precise positioning, its operational cost can be very expensive and may not be suitable for a relatively small driving environment. Alternatively, a Simultaneous Localization And Mapping (SLAM) algorithm can be used to create a point cloud map from recorded LiDAR scans. Commonly used open-source SLAM implementations include [hdl-graph-slam](https://github.com/koide3/hdl_graph_slam) (LiDAR, IMU\*, GNSS\*), [LeGO-LOAM](https://github.com/facontidavide/LeGO-LOAM-BOR) (LiDAR, IMU\*), [LeGO-LOAM-BOR](https://github.com/RobustFieldAutonomyLab/LeGO-LOAM) (LiDAR), [LIO-SAM](https://github.com/TixiaoShan/LIO-SAM) (LiDAR, IMU, GNSS), and [lidarslam-ros2](https://github.com/rsasaki0109/lidarslam_ros2) (LiDAR, IMU\*). The required sensor data for each algorithm is specified in the parentheses, where an asterisk (\*) indicates that such sensor data is optional. Most of these algorithms already have a built-in loop-closure and pose graph optimization. However, if the built-in, automatic loop-closure fails or does not work correctly, you can use [Interactive SLAM](https://github.com/SMRT-AIST/interactive_slam) to adjust and optimize a pose graph manually. For supported LiDAR models, please check the Github repository of each algorithm.
+
+It is important to note that apart from [lidarslam-ros2](https://github.com/rsasaki0109/lidarslam_ros2), these open-source SLAM implementations are based on ROS 1. Therefore, it could be problematic if you want to install them on the same machine that runs Autoware, which is based on ROS 2. To avoid this problem, you can use Docker or install them on a different machine. Another problem is the ROSBAG version, as many of these SLAM implementations require ROSBAG 1 instead of ROSBAG 2 used by Autoware. For the ROSBAG version problem, you may use this [stand-alone converter](https://gitlab.com/MapIV/rosbags) to convert a ROSBAG 2 file to a ROSBAG 1 file and vice versa.
+
+If you prefer proprietary software that is easy to use, you can try a fully automatic mapping tool from [MAP IV, Inc.](https://www.map4.jp/), [_MapIV Engine_](https://www.map4.jp/map4_engine_en/). They currently provide a trial license for Autoware users free of charge.
+
+### Vector map (Road network map)
+
+Another type of map used in Autoware is a vector map. A vector map or road network map is required for route planning, traffic light detection, and other traffic participants' trajectory prediction. Specifications of an Autoware-compatible road network map are as follows:
```

Comment:
I'm not sure but the word "vector map" and "road net map" sound like a different meaning. The latter is more rough map? 
IMO, it should unify the word  "vector map" if "road net map" is not commonly used.

Reply:
I think “road network map” is actually a more common term compared to “vector map” when people refer to a high-definition map that has details of lanes, road boundaries, junctions, etc.

As far as I understand, “vector map” is a format name. It is a proprietary format from AISAN used in [Autoware.ai](http://autoware.ai/) (ROS1 Autoware) to represent a road network map. In the current Autoware (ROS2 Autoware), the chosen format is Lanelet2. 

I think the term “vector map” is equivalent to “Lanelet2”, “OpenDrive”, or “OpenStreetMap”. They are all formats used to describe a road network.

Because of this, I thought it would be more clear to just use both words. However, if it seems to cause confusion here, maybe I can modify the text a little bit to make it clearer.